src/Evolution/Systems/NewtonianEuler/Subcell/PrimitiveGhostData.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/VariablesTag.hpp"
+#include "Evolution/Systems/NewtonianEuler/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace NewtonianEuler::subcell {
+/*!
+ * \brief Computes the mass density, velocity, and pressure on the subcells so
+ * they can be used for reconstruction.
```

Comment:
I'm a bit confused by what's going on here. Things I think would help:
- remove the "Ghost" from the class name or explain what's "ghostly" about it in the documentation
- clarify the sentence below (lines 17-18) as to which grid the copy is happening on. Is this subcell grid -> subcell grid?

Reply:
- this sends what the neighbor would consider "ghost cells" so ghost is the appropriate name. Do you have a suggestion for what to add to the documentation? I'm not sure without going into a full-blown explanation of how FD/FV works, which seems a bit overkill here.